test/utils/utils.hpp

```diff
@@ -3,28 +3,27 @@
 // SPDX-License-Identifier: Apache-2.0
 #pragma once
 
-#include <cstdint>
-#include <string>
+#include <evmc/hex.hpp>
 
 using bytes = std::basic_string<uint8_t>;
 using bytes_view = std::basic_string_view<uint8_t>;
 
-/// Encode a byte to a hex string.
```

Comment:
Nit: the others kept the docs, except this one.

Reply:
Fixed.